### ISSUE TITLE
Cyrillic letters are not applicable for query method as arguments

### ISSUE THREAD

**Issue by AlexZeDim** [lukemnet/blizzapi#51]
Hi! 

I just installed your library instead of `Blizzard.js` that is seems no longer maintenanced. So it works fine, until you try to request anything via `query` method with arguments that consist cyrillic letters. Even if I use `encodeURI` method.

You could try it yourself, with the code like this:

```
        const api = new BlizzAPI({
            region: 'eu',
            clientId: blizzardKeys._id,
            clientSecret: blizzardKeys.secret
        });
        let uri = encodeURI(`/wow/character/${arg_realm}/${arg_name}`);
        const data = await api.query(uri);
        console.log(data);
```

where `realm` is `гордунни`, and `name` equals `инициатива` or anything like this.

And I also checked the [docs ](https://blizzapi.lukem.net/docs/usage/query.html)for `query` method, and haven't found anything that helps me to understand how to request a specific `fields` for `character` or any other endpoint that supports it.

**Comment by lwojcik** [comment]
Hi! Thanks for reaching out. I'll check this and get back to you.

**Comment by lwojcik** [comment]
Fixed in v1.1.1. Please let me know if that fixes your problem.
